nvflare/app_opt/xgboost/histogram_based_v2/runners/xgb_client_runner.py

```diff
@@ -166,13 +178,13 @@ def run(self, ctx: dict):
                 lib_name = f"lib{xgb_plugin_params[PLUGIN_KEY_NAME]}.{lib_ext}"
                 xgb_plugin_params[PLUGIN_KEY_PATH] = str(get_package_root() / "libs" / lib_name)
 
-            self.logger.info(f"XGBoost secure training: {self._training_mode} Params: {xgb_plugin_params}")
+            self.logger.info(f"XGBoost secure training: split_mode: {self._split_mode} Params: {xgb_plugin_params}")
```

Comment:
minor stuff, we already printed split_mode in line 139

Reply:
Removed.